srcpkgs/tuptime/srcpkgs/tuptime/INSTALL

```diff
@@ -0,0 +1,6 @@
+# Initialize tuptime db
+case ${ACTION} in
+post)
+        exec chpst -u _tuptime /usr/bin/tuptime -q
```

Comment:
If this is the same command as when the service is run, 

1. Why does it need to happen at install time, why can't it just happen on first run?
2. Does it exit? Either this will hang until killed, or the runit service will just flap.

Reply:
1. Because it needs to create the db file with the right user (_tuptime). Leaving to the first execution could be too late becase root could execute tuptime and create the db with their user.

2. Yes, it exits